src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,44 @@
+.stopwatch {
+  @include center;
```

Comment:
The mixin `center` is not defined in this file. Make sure to define it in your mixins file and import it at the top of this file.